simapp/app.go

```diff
@@ -158,7 +157,7 @@ func NewSimApp(
 	}
 
 	// init params keeper and subspaces
-	app.ParamsKeeper = params.NewKeeper(app.cdc, keys[params.StoreKey], tkeys[params.TStoreKey])
```

Comment:
Would it not be better to keep the Codecs but maybe make them an interface so we can future proof against further experimentation?